### DISCUSSION TITLE
Custom position of the Modal

### DISCUSSION THREAD

**Discussion by HabibRh26** [NG-ZORRO/ng-zorro-antd#7978] [Q&A]
How can i set the position of the modal at the top right of the screen ?

**Comment by PishoLu** [comment]
.ant-modal has margin: 0 auto; top: 100;
use margin-left: auto; and adjust top: 0

**Comment by vanquan2002** [comment]
So, how can i set the position of the modal at the left bottom of the screen ? help me please!
